lock/dynamolock/dynamolock.go

```diff
@@ -23,17 +23,17 @@ import (
 
 type DynamoLock struct {
 	TableName    string
-	LockClient   *dynamolock.Client
+	lockClient   *dynamolock.Client
 	LockedItem   *dynamolock.Lock
 	Key          string
 	DynamoClient dynamodbiface.DynamoDBAPI
-	Options      LockOptions
+	Options      Options
 }
```

Comment:
Since you changed lockClient to be private, we were trying to come up with any use case for the various lock struct members being public, and debugging/logging was the only one that came to mind.  Can you think of any reason to keep the others public?  We could add a String() method to the Locker interface to support debugging...